subprojects/docs/src/docs/userguide/migrating_from_maven.adoc

```diff
@@ -77,50 +85,200 @@ $ gradle build
+}
+----
+<1> Attaches version 1.2.12 of Log4J to the `implementation` configuration (scope)
+
+The string identifier takes the form "__<groupId>__:__<artifactId>__:__<version>__", although Gradle refers to them as "group", "module" and "version".
+
+The above example raises an obvious question: what is that `implementation` configuration?
+It's one of the standard dependency configurations provided by the <<java_plugin#tab:configurations,Java Plugin>> and is often used as a substitute for Maven's default `compile` scope.
+
+Several of the differences between Maven's scopes and Gradle's standard configurations come down to Gradle distinguishing between the dependencies required to build a module and the dependencies required to build a module that depends on it.
+Maven makes no such distinction, so published POMs typically include dependencies that consumers of a library don't actually need.
+
+Here are the main Maven dependency scopes and how you should deal with their migration:
+
+`compile`::
+Gradle has two configurations that can be used in place of the `compile` scope: `implementation` and `api`.
+The former is available to any project that applies the Java Plugin, while `api` is only available to projects that specifically apply the <<java_library_plugin#java_library_plugin,Java Library Plugin>>.
+
++
+In most cases you should simply use the `implementation` configuration, but read the section on <<building_java_projects#sec:building_java_libraries,Building Java libraries>> and the chapter on the Java Library Plugin to find out whether you should use `api` instead for a given dependency.
+
+`runtime`::
+Use the `runtimeOnly` configuration.
+
+`test`::
+Gradle distinguishes between those dependencies that are required to _compile_ a project's tests and those that are only needed to _run_ them.
++
+Dependencies required for test compilation should be declared against the `testImplementation` configuration.
+Those that are only required for running the tests should use `testRuntimeOnly`.
+
+`provided`::
+Use the `compileOnly` configuration.
++
+Note that the <<war_plugin#sec:war_dependency_management,War Plugin>> adds `providedCompile` and `providedRuntime` dependency configurations.
+These behave slightly differently from `compileOnly` and simply ensure that those dependencies aren't packaged in the WAR file.
+However, the dependencies are included on runtime and test runtime classpaths, so use these configurations if that's the behavior you need.
+
+`import`::
+The `import` scope can be used within both `<dependencyManagement>` and `<dependencies>` blocks.
+If you're dealing with the former scenario, read the section on <<migmvn:using_boms,Using bills of materials>>.
++
+The latter scenario has no equivalent in Gradle.
+You can specify a regular dependency on the POM or its library, but the dependencies declared in the `<dependencies>` block will be treated as normal transitive dependencies of the build.
```

Comment:
`on the POM or its library` is a bit confusing imo. Maybe:
`However, in Gradle you can define dependencies on libraries but also on POM-only publications like groovy-all. In both cases the dependencies declared in the <dependencies> block of the POM will be treated as transitive dependencies of the build. Thus having a similar effect as <import>`. (<- is that true? I noticed that we never get back to explaining how this now relates to <import>)

Reply:
Actually, I may remove this. I don't know what the precise behaviour of Maven is in this case. It appears to replace `import` with `compile`, but that's only in Andy's example. I'll probably talk about declaring a dependency on a POM-only publication without reference to `import` scope within a `<dependencies>` block.